components/track/TrackAddToPlaylist.vue

```diff
@@ -0,0 +1,58 @@
+<script lang="ts" setup>
+import { TrackCollectionApi } from "@bcc-code/bmm-sdk-fetch";
+import type { TrackCollectionIdPostRequest } from "@bcc-code/bmm-sdk-fetch";
+
+const { t } = useI18n();
+
+const props = defineProps<{
+  trackId: number;
+}>();
+const emit = defineEmits<{
+  close: [];
+}>();
+
+const { data: playlists } = usePrivatePlaylists();
+const selectedPlaylistId = ref(0);
+const idsToLink = (...ids: number[]) => ids.map((id) => `</track/${id}>,`);
+
+const selectList = (playlistId: number) => {
+  selectedPlaylistId.value = playlistId;
+  const options: TrackCollectionIdPostRequest = {
+    id: playlistId,
+    link: idsToLink(props.trackId),
+  };
+  new TrackCollectionApi().trackCollectionIdPost(options);
```

Comment:
I'd rather hide this logic in a composable for the playlist. This way, we could also move the function `idsToLink` there, so this component can fully focus on the things related to what's visible and not about the API connection.

Reply:
Fixed in https://github.com/bcc-code/bmm-web/pull/293, also added better error handling.